api/v1beta1/tobiko_types.go

```diff
@@ -18,238 +18,160 @@ package v1beta1
 
 import (
 	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
-	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Hash - struct to add hashes to status
-/*
-type Hash struct {
-	// Name of hash referencing the parameter
-	Name string `json:"name,omitempty"`
-	// Hash
-	Hash string `json:"hash,omitempty"`
-}
-*/
-
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // TobikoSpec defines the desired state of Tobiko
 type TobikoSpec struct {
-	// +kubebuilder:validation:Required
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	// +kubebuilder:default="local-storage"
-        // StorageClass used to create PVCs that store the logs
-	StorageClass string `json:"storageClass"`
-
-        // +kubebuilder:validation:Optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	// This value contains a nodeSelector value that is applied to test pods
-	// spawned by the test operator.
-	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+	CommonOptions              `json:",inline"`
 
 	// +kubebuilder:validation:Optional
-        // +kubebuilder:default=false
```

Comment:
I think it is good idea to fix the lint issues. Can we make it a separate commit as part of this PR (since this is not related to the `common.go` effort)?